content/blogs/newsletter-104.md

```diff
@@ -0,0 +1,124 @@
+In fact, some of the existing core developers became inactive for quite a while, some of whom were maintainers of important core modules of the project.
+
+This left the development workflow in a state of paralysis for some of the module components of ReactOS, as nobody else in the team had the right skills to determine the correctness of the patches provided
+by contributors in order to get them shipped in the [master](https://github.com/reactos/reactos) branch.
+The ReactOS Team is currently adopting an approach to circumvent this problem, by enrolling contributors into the development team who have contributed useful patches for at least a year, have good coding skills, have collaborated with project team members and are willing to work in the area in which they are interested.
+
+With that said, the ReactOS Team is pleased to announce the addition of 4 new developers for the project -- Carl J. Bialorucki (cbialorucki), Doug Lyons (Doug-Lyons), Oleg Dubinskiy (oleg-dubinskiy) and Whindmar Saksit (whindsaks).
+Their main focus is to maintain the Shell and Audio components, as well as on any other module that requires assistance.
+
+## Symmetric Multiprocessing (SMP) support status & GCC 13 preparations
+Symmetric multiprocessing support for the AMD64 architecture is being extensively improved thanks to Timo Kreuzer (tkreuzer).
+In particular, he has implemented support for processor freezing and switching ([PR #6581](https://github.com/reactos/reactos/pull/6581)) so that every logical processor is frozen or switched properly.
+In addition, he implemented basic support for IPIs (inter-processor interrupts) which are needed for SMP scheduling, debugging, and more ([PR #6130](https://github.com/reactos/reactos/pull/6130)).
+These are only limited to APC (Asynchronous Procedure Calls), DPC (Deferred Procedure Calls) and freeze IPIs for the time being.
+
+Another important improvement for SMP is the kernel initialization in the AMD64 build architecture, which was revamped by Timo ([PR #6110](https://github.com/reactos/reactos/pull/6110)).
+This allows SMP to work properly.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-104/blender_smp.png" src="/img/blogs/newsletter-104/blender_smp.png" caption="Blender running in ReactOS 64-bit with 2 cores">}}
+{{< /gallery >}}
+
+Timo also helped fix some bugs to be able to compile ReactOS in the GCC 13 build environment ([PR #6824](https://github.com/reactos/reactos/pull/6824)).
+Much of the thanks for preparing ReactOS to build on GCC 13 goes to Simone Mario Lombardo (simonelombardo) for making ReactOS bootable.
+
+In the meantime, Justin Miller (The_DarkFire) is focusing on the x86 architecture of the SMP area.
+He worked on setting up application processor bootup ([PR #5879](https://github.com/reactos/reactos/pull/5879)) in x86 SMP HAL and is planning to further improve x86 SMP.
+On top of all of this experiments have begun to allow ReactOS to make use of the new SMP aware HAL.
```

Comment:
Suggestion:
On top of all of this, experiments have begun to allow ReactOS to make use of the new SMP aware HAL.